src/ResourceManager/Advisor/Commands.Advisor/Commands.Advisor.csproj

```diff
@@ -0,0 +1,98 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\..\..\tools\Common.Dependencies.targets" />
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{3EC36A66-9B32-4D15-A6FC-5804AA15D52C}</ProjectGuid>
```

Comment:
you can delete this entire file.

Reply:
What do you mean by this.. I would like to have additional information about how the new feature works with out using this file.

As this contradicts the setup guidelines outlined. I use the Advisor.sln to run the VS.